src/Symfony/Component/Console/Tester/CommandTester.php

```diff
@@ -24,9 +24,12 @@ class CommandTester
 {
     use TesterTrait;
 
+    private Command $command;
+
     public function __construct(
-        private Command $command,
+        object $command,
```

Comment:
Changing the type is allowed here, but you can use a more specific type:
```suggestion
        callable|Command $command,
```

Reply:
Done